W-12679396 - docs - Exchange contributor cannot delete version (#278)
mulesoft/docs-access-management: modules/ROOT/pages/roles.adoc

* **Exchange Contributors**: This role has the following permission scopes:
+
** Create content

> ** Manage own content, including the ability to add a new version, edit your own version content, but can not delete own version.

Reviewer: ```suggestion
** Manage own content, including the ability to add a new version and edit your own version content, but you cannot delete your own version.
```
